docs/topics/minishift-install-prereq.adoc

```diff
@@ -13,6 +13,19 @@ $ minishift version
 Minishift version: 1.0.0
 ----
 
+. Determine the command to install the correct version of the `oc` binary and run the command. 
```

Comment:
Actually with this paragraph added, the WARNING below (line 40) seems no longer necessary, or it could be shortened and moved to this section?